Kitodo/src/main/resources/kitodo_config.properties

```diff
@@ -541,6 +541,12 @@ showOcrButton=false
 # Base path to OCR, without parameters
 ocrUrl=
 
+# -----------------------------------
+# Display process properties
+# -----------------------------------
+
+# Process properties to be displayed in process list
+processProperties=
```

Comment:
The name is not self-explanatory enough
processPropertiesForProcessList 
or something

Reply:
Would `processPropertyColumns` be ok?